GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs

```diff
@@ -10,9 +10,12 @@ public partial class DiffViewerSettingsPage : SettingsPageWithHeader
 
 - For a single selected commit, show the difference with its parent commit.
 - For a single selected merge commit, show the difference with all parents.
-- For two selected commits, show the difference between the commits as well as the difference between a common ancestor (BASE) to the last selected commit.
+- For two selected commits with a common ancestor (BASE):
+ - Show the difference between the commits.
+ - The difference of unique files from BASE to each of the selected commits.
+ - The difference of common files (identical changes) from BASE to the commits.
```

Comment:
A little more indentation would make this nesting more visually obvious, especially in a non-monospace font. I didn't see it at first in the screenshot.

```suggestion
- For two selected commits with a common ancestor (BASE):
   - Show the difference between the commits.
   - The difference of unique files from BASE to each of the selected commits.
   - The difference of common files (identical changes) from BASE to the commits.
```